lib/pbench/server/indexer.py

```diff
@@ -3184,6 +3181,17 @@ def __init__(self, idxctx, username, tbarg, tmpdir, extracted_root):
         mtime = datetime.utcfromtimestamp(tb_stat.st_mtime)
         self.tb = tarfile.open(self.tbname)
 
+        # Build a map showing the documents in each Elasticsearch index so we
+        # can find them later to UPDATE or DELETE without searching all
+        # indices.
+        #
+        # {
+        #     "jam-pbench.v6.run-data.2021-05": "<doc-id>",
+        #     "jam-pbench.v6.run-toc.2021-05": "<doc-id>",
```

Comment:
The value of the map keys is a list, because we scatter multiple documents across several indices, including the TOC. I missed updating the comment.

Reply:
Fixed.